### ISSUE TITLE
Question: mouseover casting

### ISSUE THREAD

**Issue by layday** [tullamods/Dominos#379]
Hey, thanks for Dominos - I recently switched from Bartender and one thing I miss is the ability to enable mouseover casting for all buttons in a bar, which would spare me the trouble of having to create mouseover macros or configure Clique.  Is there a way to accomplish something similar in Dominos?

**Comment by Tuller** [comment]
Not currently, no

**Comment by damiantymkowski** [comment]
@Tuller Has this feature been added? Is there an add-on that makes this possible? I like Dominos and don't want to go back to ElvUI.

**Comment by Tuller** [comment]
No, not yet. This is more wrapped up in me implementing settings on a per bar level.

**Comment by Tuller** [comment]
This now works
`/run Dominos.Frame:Get(1):SetUnit("mouseover")`

Running this will return to the normal behavior:
`/run Dominos.Frame:Get(1):SetUnit("none")`
